src/welcome/admin.js

```diff
@@ -766,24 +767,24 @@ const EditorSettings = props => {
 								} }
 								help={ __( 'Adds a toolbar button for advanced copying and pasting block styles', i18n ) }
 							/>
-							<AdminToggleSetting
+							{ settings.stackable_enable_block_defaults && <AdminToggleSetting
```

Comment:
@mxkae I believe this setting isn't for the block defaults, but instead is for our block layouts feature.